src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/MatchType.java

```diff
@@ -5,6 +5,7 @@ public enum MatchType {
     NAME_DOB_PNC,
     /** Matches to all the parameters supplied but at least one from the one of the aliases associated to the offender */
     NAME_DOB_ALIAS,
+    ALL_SUPPLIED_ALIAS,
```

Comment:
We can use the existing NAME_DOB_ALIAS for these records - there's technical reasons why we have to map the value returned by Offender Search to these values. I can't remember the specifics off the top of my head but will refresh my memory and update you when I've done so, or put it on the matcher ticket.

Reply:
The code that does the mapping is [here](https://github.com/ministryofjustice/court-case-matcher/blob/main/src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/offendersearch/OffenderSearchMatchType.java)

So the reason we do that mapping is because the value that Offender Search returns `ALL_SUPPLIED` doesn't tell us what it was that we supplied, e.g. name, dob pnc. It matters to us whether we've matched on PNC as this is considered to be an authoritative match - if it's just name and dob then that's not as high quality so we introduced this mapping to retain that information.

Reply:
addressed